### ISSUE TITLE
Feature Request - define Serialport in configuration.h

### ISSUE THREAD

**Issue by cocktailyogi** [repetier/Repetier-Firmware#360]
instead of hal.h

This not a first priority-guess, but it saves nerves.

For performance reasons, I am using "SerialUSB". For every update I have to set RFSERIAL in hal.h. I guess, this setting better belong into config-file.

What do you think?

**Comment by repetier** [comment]
Yes, that is where i finally want it. Just need some more brainstorming on how to do it together with bluetooth option and online config tool. Always getting more options with each update :-)
